src/tools/ToolBoxSelect.ts

```diff
@@ -0,0 +1,24 @@
+import { ToolConstructor } from "./BaseTool";
+import { ToolSelectMove } from "./ToolSelectMove";
+import { GlobalStore } from "@/store";
+
+/**
+ * Enables Selection, Moving, and Panning via base class
+ * This Tool creates a rectangle as a lasso.
+ */
+const ToolBoxSelect: ToolConstructor = class ToolBoxSelect extends ToolSelectMove {
+  onNewMouseMoveSelection(point: [number, number]): void {
+    if (this.selectionLassoStart === null) {
+      return;
+    }
+    GlobalStore.commit("setSelectionLasso", [
```

Comment:
Ah, I see, so you're drawing a rectangle using `polyline`! Nice job keeping things re-usable!